utils/similarity/cache.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
Consider 
```
try:
  import numpy as np
except ImportError:
  ## print "please install numpy: pip ...
```

Reply:
Note: import numpy is present in other files

Reply:
fastText is a requirement for this util, and it checked in the initialization:

https://github.com/crytic/slither/blob/dev-simil/utils/similarity/__init__.py#L4-L12

And numpy is a requirement of fastText, so adding that try/except will have no effect. I can still do it if you want to keep the consistency.

Reply:
I finally added this try/catch, in case of the user decides to import this module individually (which is very unlikely, but it could happen)